commands-yml/commands/context/get-contexts.yml

```diff
@@ -8,7 +8,8 @@ description:
     For information on the format of the context names, see the [get context
     documentation](/docs/en/commands/context/get-context.md).
 
-    On iOS, using the XCUITest driver, one can use the `mobile: getContexts`
+    On iOS with XCUITest driver and Android with UIAutomator2/Espresso driver,
+    one can use the `mobile: getContexts` 
```

Comment:
it would be nice to address a bit more.

e.g

> `mobile: getContexts` is available on iOS (XCUITest) and Andoid (UIAutomator2 and Espresso) to get more detaled contexts.

Reply:
I've updated. Does it meet your expectation? It looks simpler while I did drop some info though.